modules/ROOT/pages/split-aggregate-scope.adoc

```diff
@@ -0,0 +1,109 @@
+For example:
+
+----
+<set-variable variableName="var1" value="var1"/>
+<set-variable variableName="var2" value="var2"/>
+<split-aggregate>
+	<collection>
+		#[['apple', 'banana', 'orange']]
+	</collection>
+	<choice>
+		<when expression="#[payload == 'apple']">
+			<set-variable variableName="var2" value="newValue"/>
+			<set-variable variableName="var3" value="val1"/>
+
+		</when>
+		<when expression="#[payload == 'banana']">
+			<set-variable variableName="var3" value="val2"/>
+		</when>
+		<otherwise>
+			<set-variable variableName="var3" value="val3"/>
+			<set-variable variableName="var4" value="val4"/>
+		</otherwise>
+	</choice>
+</split-aggregate>
+<!-- After aggregation, the variables will be: 
+{var1: "var1", var2: "newValue", var3: ["val1, val2, val3"], var4: "val4"} -->
+----
+
+== Error Handling
+
+When an error is thrown in one route, since every route is being processed in parallel, Split Aggregate will continue t process each route and then invoke the xref:intro-error-handlers.adoc[Error Handler], with the aggregation of the results and errors of all the routes. For example:
+
+----
+<split-aggregate collection="#['banana', 'apple']">
+	<choice>
+		<when expression="#[payload == 'banana']">
+			<!-- Processor that throws exception -->
+		</when>
+		<otherwise>
+			<set-payload value="#[payload ++ '-result']"/>
+		</otherwise>
+	</choice>
+</split-aggregate>
+<error-handler>
+	<on-error-continue type="COMPOSITE_ROUTING">
+		<!-- This will have the error thrown by the above processor --> 
+		<logger message="#[error.errorMessage.payload.failures['0']"/>
+		<!-- This will be a null value --> 
+		<logger message="#[error.errorMessage.payload.failures['1']"/>
+		<!-- This will be a null value --> 
+		<logger message="#[error.errorMessage.payload.results['0']"/>
+		<!-- This will have the result of this (correctly executed) route --> 
+		<logger message="#[error.errorMessage.payload.results['1']"/>
+	</on-error-continue>
+</error-handler>
+----
+
+=== Differences between Split Aggregate and For Each scopes
+
+There is an obvious similarity between the Split Aggregate and the xref:for-each-scope-concept.adoc[For Each]: both split the defined collection and process each element in it with the components within the scope. Also, in both cases, each route runs with the same initial context.
+The difference with these two scopes are:
+
+1. For Each works sequentially, while the Split Aggregate does it in parallel.
+1. The For Each does not modify the payload, while the Split Aggregate will output a collection of the output messages of each iteration.
+1. Error handling: since the first mentioned difference, For Each execution will be interrupted when an error is raised (and the Error Handler will be invoke), while Split Aggregate will work as defined before, processing every route before invoking the Error Handler with a `MULE:COMPOSITE_ROUTE` error type.
```

Comment:
```suggestion
1. Error handling: since the first mentioned difference, For Each execution will be interrupted when an error is raised (and the Error Handler will be invoked), while Split Aggregate will work as defined before, processing every route before invoking the Error Handler with a `MULE:COMPOSITE_ROUTE` error type.
```